specification/cognitiveservices/Language.AnalyzeConversations-authoring/models.tsp

```diff
@@ -378,7 +379,7 @@ model InnerErrorModel {
 /**
  * Represents the options used to create or update a project.
  */
-@clientName("CreateProjectOptions", "csharp")
+@clientName("CreateProjectConfig", "csharp")
```

Comment:
A "Config" suffix is not common in .NET. Who had you make these changes for C#?

Reply:
Thanks for your comment, heaths. This was recommended by the pipeline azure-sdk-for-net-track2 for automatically generate c# sdk. It recommended me to change the suffix from "Options" to "Config".

Reply:
Otherwise, the pipeline will fail.

Reply:
Hi @heaths , thank you for your advice! I have made all the changes based on your suggestions, except for this "Config" suffix. The pipeline azure-sdk-for-net-track2 recommend this suffix and doesn’t allow "Options" to pass. Do you have any suggestions for the naming here? Thank you!

Reply:
Hi @heaths , I just try to run the pipeline with "Options" suffix, the issue still exist, showing the error:
error AZC0030: Model name 'CreateProjectOptions' ends with 'Options'. Suggest to rename it to 'CreateProjectConfig' or any other appropriate name. (The link is at the end)

If "Config" suffix is not recommended, can we change it to "Parameters" here?
Thank you!

https://github.com/Azure/azure-rest-api-specs/pull/31360/checks?check_run_id=33878682485#:~:text=src%0Acmdout%09%5B.Net%5D-,/mnt/vss/_work/1/s/azure%2Dsdk%2Dfor%2Dnet/sdk/cognitivelanguage,Conversations.Authoring/src/Azure.AI.Language.Conversations.Authoring.csproj%3A%3ATargetFramework%3Dnetstandard2.0%5D,-cmdout%09%5B.Net%5D%20/mnt

Reply:
Please contact @KrzysztofCwalina and @tg-msft, the .NET architects. I don't know why they changed their recommendation, but it was mostly "Options" and, for a short time, "Parameters".

Reply:
Thank you, heaths. Will reach out to them to ask.

Reply:
this type is not an input model. this is why the analysis check flags it.

Reply:
@KrzysztofCwalina Thank you for your advice!

Reply:
Hi @heaths , I have changed all "Config" suffix to "Details" as KrzysztofCwalina and Ted suggested. Is there more changes I need to make? Thank you!